talemate_frontend/src/components/AppConfigPresets.vue

```diff
@@ -52,7 +52,7 @@
                             <v-slider thumb-label="always" density="compact" v-model="config.inference[selected[0]].frequency_penalty" min="0.1" max="1.0" step="0.1" label="Frequency Penalty"></v-slider>
                             <v-slider thumb-label="always" density="compact" v-model="config.inference[selected[0]].repetition_penalty" min="0.1" max="2.0" step="0.1" label="Repetition Penalty"></v-slider>
                             <v-slider thumb-label="always" density="compact" v-model="config.inference[selected[0]].repetition_penalty_range" min="0" max="1024" step="256" label="Repetition Penalty Range"></v-slider>
```

Comment:
I usually keep my rep penalty range around 2000-2800, I'd recommend increasing the max value

Reply:
One more thing to add, I would include `temperature_last` as a checkbox and `min_p` with a minimum value of 0 which disables it. `min_p` doesn't work correctly unless `temperature_last` is enabled and in some cases you might not want to have `min_p` or `temperature_last` enabled. I would also make `min_p` in increments of 0.05 as it's a pretty sensitive sampler.

Reply:
thanks, pushed those changes